frontend/src/metabase/admin/settings/components/SettingsPages/UpdatesSettingsPage.tsx

```diff
@@ -0,0 +1,74 @@
+import cx from "classnames";
+import { c, t } from "ttag";
+
+import { UpsellHostingUpdates } from "metabase/admin/upsells";
+import { useSetting } from "metabase/common/hooks";
+import CS from "metabase/css/core/index.css";
+import { Box, Flex } from "metabase/ui";
+
+import { AdminSettingInput } from "../widgets/AdminSettingInput";
+import { VersionUpdateNotice } from "../widgets/VersionUpdateNotice";
+
+export function UpdatesSettingsPage() {
+  const isHosted = useSetting("is-hosted?");
+  const checkForUpdates = useSetting("check-for-updates");
+  return (
+    <Flex justify="space-between" data-testid="settings-updates">
+      <div style={{ width: "585px" }}>
+        {!isHosted && (
+          <ul>
```

Comment:
`<ul>` seems wrong here? there's no list.

Reply:
The old settings widgets were automatically laid out in a list, but it seems to be an abuse of what a list is.

Reply:
yeah, let's stop the abuse 🛑